### PR TITLE
Add IAM example to add users to a project

### PR DESCRIPTION
**Pull request by salrashid123**
Supplemental stand-alone  example to add users to a project.

### CODE REVIEW

**Comment by salrashid123** [comment]
Can we start the eval/review for this?

/cc @krwenholz @jaivasanth-google

**Comment by krwenholz** [comment]
@adamharwayne who owns the internal bug for this.

**Comment by salrashid123** [comment]
Refactored to community folder; moved PR specific instructions to folder

File: community/iam/project/iam_project.jinja [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+
+- name: set-iam-policy-delete
+  action: gcp-types/cloudresourcemanager-v1:cloudresourcemanager.projects.setIamPolicy
+  metadata:
+    runtimePolicy:
+    - DELETE
+  properties:
+    resource: {{ env['project'] }}
+    policy: $(ref.get-iam-policy)
```
Comment (adamharwayne) [inline community/iam/project/iam_project.jinja]: DM no longer supports delete actions referring to other delete actions (they were buggy and we have disabled it until we iron out the bugs).

**Pull request closed without merging**